signer/tests/integration/setup.rs

```diff
@@ -400,6 +401,108 @@ pub async fn backfill_bitcoin_blocks(db: &PgStore, rpc: &Client, chain_tip: &bit
     }
 }
 
+pub async fn fill_signers_utxo<R: rand::RngCore + ?Sized>(
+    db: &PgStore,
+    bitcoin_block: model::BitcoinBlock,
+    aggregate_key: &PublicKey,
+    mut rng: &mut R,
+) {
```

Comment:
Hmmm, a similar method should be added to the `TestSweepSetup` struct, since that generates a UTXO for the signers to use. But we don't have methods for writing to all of the necessary tables there yet.

Reply:
Well after the base branch gets merged in we can use the new test setup struct.